trajopt_optimizers/trajopt_optimizers_test_suite/src/joint_velocity_optimization_unit.cpp

```diff
@@ -0,0 +1,55 @@
+#include <trajopt_utils/macros.h>
+TRAJOPT_IGNORE_WARNINGS_PUSH
+#include <gtest/gtest.h>
+#include <iostream>
+
+#include <ifopt/problem.h>
+#include <ifopt/ipopt_solver.h>
+#include <console_bridge/console.h>
+TRAJOPT_IGNORE_WARNINGS_POP
+
+#include <trajopt_sqp/trust_region_sqp_solver.h>
+#include <trajopt_sqp/osqp_eigen_solver.h>
+#include <trajopt_ifopt/constraints/joint_position_constraint.h>
+#include <trajopt_ifopt/constraints/joint_velocity_constraint.h>
+#include <trajopt_ifopt/variable_sets/joint_position_variable.h>
+#include <trajopt_ifopt/costs/squared_cost.h>
+
+#include <joint_velocity_optimization_unit.hpp>
+
+/**
+@brief Joint position constraints with a squared velocity cost in between. Optimized using ipopt
+ */
+TEST_F(VelocityConstraintOptimization, velocity_constraint_optimization_ipopt)
+{
+  ifopt::Problem nlp_opt(nlp_);
+  ifopt::IpoptSolver solver;
+  solver.SetOption("derivative_test", "first-order");
+  solver.SetOption("linear_solver", "mumps");
+  //  ipopt.SetOption("jacobian_approximation", "finite-difference-values");
+  solver.SetOption("jacobian_approximation", "exact");
+  solver.SetOption("print_level", 5);
+
+  runTests<ifopt::IpoptSolver>(solver, nlp_opt);
+}
```

Comment:
This can stay in this package since it is implemented in IFOPT and thus is really testing the test suite, but lets put it in the test directory instead of src.